src/main/js/util/fieldModelUtilities.js

```diff
@@ -255,13 +255,3 @@ export function checkModelOrCreateModelWithDefaults(fieldModel, fields) {
     newModel.keyToValues = newKeyToValues;
     return newModel;
 }
-
-export function handleChange(self, target, stateName) {
```

Comment:
Was this deleted because it wasn't being used?

Reply:
This was deleted because I found a better place for it to live. The FieldsPanel which handle all the field creation and what not now handles field values changing.